cmd/cored/main.go

```diff
@@ -249,6 +249,13 @@ func main() {
 		err := server.Serve(listener)
 		chainlog.Fatalkv(ctx, chainlog.KeyError, errors.Wrap(err, "Serve"))
 	}()
+
+	// Verify that we're connected to the rest of the cluster, if initialized.
+	err = errors.Root(sdb.Ping())
+	if err == context.DeadlineExceeded {
+		chainlog.Fatalkv(ctx, chainlog.KeyError, "Unable to reach rest of raft cluster. Was the node evicted?")
```

Comment:
It would be pretty cool if we could get a definitive "you are not a member" message somehow. I guess that would take a bunch of plumbing, including looking at the http response in sendmsg. Maybe a todo for that? Or is there a better idea?